index.js

```diff
@@ -577,6 +625,17 @@ class ServerlessAppsyncPlugin {
           AwsRegion: ds.config.region || config.region,
           Endpoint: ds.config.endpoint,
         };
+      } else if (ds.type === 'RELATIONAL_DATABASE') {
+        resource.Properties.RelationalDatabaseConfig = {
+          RdsHttpEndpointConfig: {
+            AwsRegion: ds.config.region || config.region,
+            DbClusterIdentifier: ds.config.dbClusterIdentifier,
+            DatabaseName: ds.config.databaseName,
+            Schema: ds.config.schema,
+            AwsSecretStoreArn: ds.config.awsSecretStoreArn,
```

Comment:
@roznalex would it make sense to also auto-generate the secret store for the datasource?
````yml
awsSecretStoreArn: arn:...... # used if provided
secretSotreConfig: #used if awsSecretStoreArn is not provided
  username: USERNAME 
  password: COMPLEX_PASSWORD
````

If `awsSecretStoreArn` is provided, it would be used, but if not and `secretSotreConfig` is present, the plugin would do the following:
- auto-generate a [AWS::SecretsManager::Secret](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-secretsmanager-secret.html) resource
- store the given username and password in it
- resolve its ARN and inject it into `awsSecretStoreArn`


Not sure if it makes sense. 
How do you currently handle secrets? Do you generate the AWS::SecretsManager::Secret in serverless ?
The tricky part here is handling the secrets without leaking them in the yml file.

Reply:
I create the "AWS::SecretsManager::Secret" resource in a Cloudformation template where I pass all the secrets through environment variables.

```yaml  RDSClusterSecret:
    Type: "AWS::SecretsManager::Secret"
    Properties:
      Name: "rds-cluster-secret-name"
      Description: RDS cluster credentials
      SecretString: '{"username":"${self:custom.environment.RDS_USERNAME}","password":"${self:custom.environment.RDS_PASSWORD}"}'```

I am not sure about the autogeneration that you mentioned above. But we might think about that.

Reply:
@roznalex 👍 
it was just an idea... it could also come in a future iteration.
We could start with a simple implementation